InstantSearch.js/routing-seo-friendly/src/app.js

```diff
@@ -1,85 +1,15 @@
 /* global instantsearch algoliasearch */
+import searchRouter from './search-router';
```

Comment:
What about `searchRouting` rather than `searchRouter` ?

It is is a bit confusing because one of the option available for `routing` is called `router`.

Reply:
Renamed!